File: connectors/connector-basecamp-classic/src/main/scala/com/taskadapter/connector/basecamp/classic/BasecampClassicField.scala
package com.taskadapter.connector.basecamp.classic

import com.taskadapter.model._

object BasecampClassicField {
  val content = Field("Content")
  val fields = List(content, DoneRatio, DueDate, AssigneeFullName, CreatedOn, UpdatedOn, ClosedOn)
}

File: connectors/connector-basecamp-classic/src/main/java/com/taskadapter/connector/basecamp/classic/BasecampClassicField.java
package com.taskadapter.connector.basecamp.classic;

import com.taskadapter.model.AllFields;
import com.taskadapter.model.CustomString;
import com.taskadapter.model.Field;

import java.util.List;

public class BasecampClassicField {

    public static final Field<String> content = new CustomString("Content");

    public static List<Field<?>> fields = List.of(
            content,
            AllFields.doneRatio,
            AllFields.dueDate,
            AllFields.assigneeFullName,
            AllFields.createdOn,
            AllFields.updatedOn,
            AllFields.closedOn);
}
